src/app/services/game-play.service.ts

```diff
@@ -0,0 +1,99 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, fromEvent } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+import { AnswerService } from './answer.service';
+import { Answer } from '../models/answer';
+
+interface Game {
+  answer: Answer;
+  acceptedGuesses: string[];
+  status: Status;
+  currentGuessIdx: number;
+  currentGuess: string;
+};
+
+enum Status {
+  playing,
+  won,
+  lost
+};
+
+const emptyGame: Game = {
+  answer: <Answer>{},
+  acceptedGuesses: [],
+  status: Status.playing,
+  currentGuessIdx: 0,
+  currentGuess: ''
+};
+
+@Injectable({
+  providedIn: 'root'
+})
+export class GamePlayService {
+  private readonly game$ = new BehaviorSubject<Game>(emptyGame);
+  constructor(private answerService: AnswerService) { this.startNewGame() }
+
+  public readonly currentGuess$ = this.game$.pipe(
+    map(game => game.currentGuess)
+  )
+  public readonly acceptedGuesses$ = this.game$.pipe(
+    map(game => game.acceptedGuesses)
+  )
+  public readonly currentGuessIdx$ = this.game$.pipe(
+    map(game => game.currentGuessIdx)
+  )
+
+  protected get game() {
+    return this.game$.getValue();
+  }
+
+  public addLetter(letter: string) {
+    if (this.game.currentGuess.length === 5) return;
+    let updatedGuess = this.game.currentGuess + letter
```

Comment:
Nit-picky: always use `const` whenever possible. Alternatively, you don't really need this variable declaration and could just do everything on line 57 in this case since there isn't a whole lot of code here